src/LobbyForm.tsx

```diff
@@ -79,26 +109,36 @@ const hostStyles = StyleSheet.create({
     fontSize: "8px",
   },
   button: {
+    padding: "0px",
+    boxSizing: "border-box",
     height: "100%",
     border: "none",
-    borderRadius: "0 8px 8px 0",
+    borderRadius: "60px",
     minWidth: "32px",
-    backgroundColor: "#EEEEEE",
+    backgroundColor: "#1D1E33",
+    ":hover": {
+      backgroundColor: "#069AB4",
+    },
+    ":active": {
+      backgroundColor: "#9CF6FC",
+    },
+  },
+  button2: {
     ":hover": {
-      backgroundColor: "#DDDDDD",
+      backgroundColor: "#7139E0",
     },
     ":active": {
-      backgroundColor: "#CCCCCC",
+      backgroundColor: "#BA9CFC",
     },
   },
   input: {
     padding: "8px",
     fontSize: "12px",
     minWidth: "160px",
     fontFamily: "Arial",
-    border: "1px solid #EEEEEE",
+    border: "none",
     borderRadius: "8px 0 0 8px",
-    "::placeholder": {
+    "input::placeholder": {

```

Comment:
can we just remove the placeholder bits? We shouldn't need to style the input as it should be already styled from the component itself (TextInput)